<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvnCommitDialog</class>
 <widget class="QWidget" name="SvnCommitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subversion</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="logGroup">
     <property name="title">
      <string>Commit Message</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="logEdit">
        <property name="toolTip">
         <string>Enter the log message.</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;Log Message&lt;/b&gt;
&lt;p&gt;Enter the log message for the commit action.&lt;/p&gt;</string>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Recent commit messages</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="recentComboBox">
        <property name="toolTip">
         <string>Select a recent commit message to use</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="changeListsGroup">
     <property name="title">
      <string>Changelists</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="changeLists">
        <property name="toolTip">
         <string>Select the change lists to limit the commit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="keepChangeListsCheckBox">
        <property name="toolTip">
         <string>Select to keep the changelists</string>
        </property>
        <property name="text">
         <string>Keep changelists</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>logEdit</tabstop>
  <tabstop>recentComboBox</tabstop>
  <tabstop>changeLists</tabstop>
  <tabstop>keepChangeListsCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
